downstream/assemblies/platform/assembly-installing-hub-operator.adoc

```diff
@@ -26,6 +26,8 @@ You can use these instructions to install the {HubName} operator on {OCP}, speci
 . Navigate to *Operators* > *Installed Operators*.
 . Locate the *Automation hub* entry, then click *Create instance*.
 
+include::platform/con_storage-options-for-operator-installation-on-ocp.adoc[leveloffset=+2]
+include::platform/proc_provision-ocp-storage-with-readwritemany-access-mode.adoc[leveloffset=+3]
```

Comment:
I like that you use underscore between the `con`, `proc`, `ref` prefix in the filenames. I'd honestly like to converge on this as standard in AAP docs (it would help with consistency).

One comment I do have here is that perhaps we can shorten the filenames by omitting things like prepositions and obvious stuff. When filenames get too long it can cause the include directive to span multiple lines, which looks yucky in an editor. Plus long filenames can actually lead to system issues because the full path, including the filename, is taken into account for the maximum number of characters.

How about something like "con_operator-storage-requirements.adoc" and "proc_provisioning-readwritemany-access-mode.adoc"?

It is good to see that the filenames closely match the titles, which is great for navigating the file system, so I'll let you work with it and figure out the best filenames. But I would recommend trimming them down a little. Cheers.

Reply:
Hi Don. I use New Doc to create all of my modularization. So it adds the underscore after the prefix, which brings up a point I meant to ask the team. I noticed a lot of our files do not follow that standard. So I needed to ask about that.

As for shortening the files and leaving out prepositions or other miscellaneous/less important words, unfortunately, New Doc takes that are added within quotes and creates the file names. It doesn't leave out any words. I do like the idea of shortening the file names. So perhaps on my next use of New Doc, I can remove the extraneous words from the title for the purpose of creating the file name and when I edit the content, I can add whatever is necessary for the title. Thanks for helping me come up with an idea to make file names more concise.